pkg/types/powervs/platform.go

```diff
@@ -4,9 +4,6 @@ package powervs
 // use.
 type Platform struct {
 
-	// ServiceInstanceID is the ID of the Power IAAS instance created from the IBM Cloud Catalog
-	ServiceInstanceID string `json:"serviceInstanceID"`
-
```

Comment:
PVSNetworkName and ClusterOSImage are also ServiceInstance specific if I recall

Reply:
As for `ClusterOSImage` I think it might be separate:
```
./pkg/asset/installconfig/powervs/platform.go:          p.ClusterOSImage = osOverride
./pkg/asset/machines/powervs/machines.go:       if platform.ClusterOSImage != "" {
./pkg/asset/machines/powervs/machines.go:               image = platform.ClusterOSImage
./pkg/asset/manifests/operators.go:                     ClusterOSImage:             p.ClusterOSImage,
./pkg/asset/rhcos/image.go:             if config.Platform.PowerVS.ClusterOSImage != "" {
./pkg/asset/rhcos/image.go:                     return config.Platform.PowerVS.ClusterOSImage, nil
./pkg/types/powervs/platform.go:        // ClusterOSImage is a pre-created Power VS boot image that overrides the
./pkg/types/powervs/platform.go:        ClusterOSImage string `json:"clusterOSImage,omitempty"`
```

Reply:
I think `PVSNetworkName` removal could be in a separate and simple PR?

Reply:
okay I'm fine with waiting on these